Userland/Libraries/LibCrypto/Curves/SECPxxxr1.h

```diff
@@ -186,6 +186,11 @@ class SECPxxxr1 : public EllipticCurve {
         auto r_bigint = TRY(asn1_decoder.read<Crypto::UnsignedBigInteger>(Crypto::ASN1::Class::Universal, Crypto::ASN1::Kind::Integer));
         auto s_bigint = TRY(asn1_decoder.read<Crypto::UnsignedBigInteger>(Crypto::ASN1::Class::Universal, Crypto::ASN1::Kind::Integer));
 
+        size_t expected_word_count = KEY_BIT_SIZE / 32;
+        if (r_bigint.length() < expected_word_count || s_bigint.length() < expected_word_count) {
+            return false;
```

Comment:
Is this a good idea? I thought that having constant time things in crypto was the way to do things.

Reply:
I belive this should be ok to do, since we need a minimum of `expected_word_count` words to have a valid key to begin with.